lib/ts/recipe/passwordless/emaildelivery/services/smtp/passwordlessLogin.ts

```diff
@@ -0,0 +1,2912 @@
+                                        <table align="center" border="0" cellpadding="0" cellspacing="0" width="100%"
+                                            class="templateContainer">
+                                            <tr>
+                                                <td valign="top" class="bodyContainer">
+                                                    <table border="0" cellpadding="0" cellspacing="0" width="100%"
+                                                        class="mcnCodeBlock">
+                                                        <tbody class="mcnTextBlockOuter">
+                                                            <tr>
+                                                                <td valign="top" class="mcnTextBlockInner">
+                                                                    <p
+                                                                        style="font-family:'Helvetica', sans-serif; margin-left: 3%; margin-right: 3%; font-size: 28px; line-height: 26px; font-weight:700; margin-bottom: 40px; margin-top: 48px; text-align: center; color: #222">
+                                                                        Login to ${appName}</p>
+                                                                </td>
+                                                            </tr>
+                                                        </tbody>
+                                                    </table>
+                                                    <table border="0" cellpadding="0" cellspacing="0" width="100%"
+                                                        class="mcnCodeBlock">
+                                                        <tbody class="mcnTextBlockOuter">
+                                                            <tr>
+                                                                <td valign="top" class="mcnTextBlockInner">
+        
+        
+                                                                    <div
+                                                                        style="background-color:#fff; margin-left: 3%; margin-right: 3%; border: 1px solid #ddd; border-radius: 6px;">
+                                                                        <div style="padding-left: 15%; padding-right: 15%;">
+        
+                                                                            <p
+                                                                                style="font-family:'Helvetica', sans-serif; font-size: 16px; line-height: 26px; font-weight:700; text-align: center; padding-top: 24px; padding-bottom: 24px; padding-left: 8%; padding-right: 8%; ">
+                                                                                Enter the below OTP in your login screen. Note
+                                                                                that the link expires in ${humaniseMilliseconds(
```

Comment:
Small optimisation: There are multiple occurences of this. You may pass it as a variable to avoid calculating it every time.

Reply:
done